lib/site_template/.gitignore

```diff
@@ -1,3 +1,4 @@
 _site
 .sass-cache
 .jekyll-metadata
+README.txt
```

Comment:
i see no reason why we wouldn't want to commit this to a git repository too, many other site* generators do it